### PR TITLE
[doc] update description for chromedriver

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Add steps for "setup test environment" in file Writing Test Cases for NW.js.md
- Update description for chromedriver in file Test with Chromedriver.md

### CODE REVIEW

**Comment by wanghongjuan** [comment]
@rogerwang I am confused about remoting tests, when running `python test/test.py -t 80 remoting`, result is `No tests to run`, is this correct? Would it will be better if replace it to `sanity` ?

**Comment by rogerwang** [comment]
Only `sanity` is used. Others are deprecated.

File: docs/For Developers/Writing Test Cases for NW.js.md [hunk 1]
```diff
@@ -11,7 +11,12 @@ Each test case in NW.js is a runnable app, so that you can manually run the case
 
 In NW.js, there are two types of test cases: `auto` and `remoting`. See [Write Test Cases](#write-test-cases) sections below to see the instructions.
 
-To run the test cases, use the following command:
+## Setup Test Environment
+
+1. After cloning this repository, please download SDK build from NW.js website, extract it, then copy all of files of its to this repository.
+2. Export CHROMEDRIVER to system environment.
```
Comment (rogerwang) [inline docs/For Developers/Writing Test Cases for NW.js.md]: PYTHONPATH should be set to third_party/webdriver/pylib/ as well.

Comment (wanghongjuan) [reply docs/For Developers/Writing Test Cases for NW.js.md]: I didn't set this path, but can run sanity tests with `python test/test.py -t 80 sanity`.

**Comment by rogerwang** [comment]
Yeah, but that isn't guaranteed to work and we have modification in that
directory. So this should be put into the document.

**Comment by wanghongjuan** [comment]
Could you please explain this more detail? Since I can't find the dir `third_party/webdriver/pylib/` in this repo, and not clear where used this setting.

**Comment by rogerwang** [comment]
> Could you please explain this more detail? Since I can't find the dir third_party/webdriver/pylib/ in this repo, and not clear where used this setting.

It could be set in the environment variable `PYTHONPATH`: `export PYTHONPATH=/path/to/src/third_party/webdriver/pylib/`

**Pull request closed without merging**